Packs/ThreatIntelligenceManagement/Scripts/IncidentsPerFeed/IncidentsPerFeed.yml

```diff
@@ -0,0 +1,37 @@
+args:
+  - default: false
```

Comment:
Where is the `from` argument that you use in your code? It's not in the YAML as a script argument.

Reply:
It's passed automatically by widgets.